features/step_definitions/journey/registration_steps.rb

```diff
@@ -0,0 +1,70 @@
+# Group pieces together when they are always consecutive (renewal - new lower tier - new upper tier)
+# Helpers vs Step to group things together - Choose helpers, avoid steps.
+# Helpers vs Step - repeat helpers, not steps.
+# Use `Given` to set up instance variables for the rest of the journey decisions
```

Comment:
Happy to explain this in the suite, but is it possible to move it to README.md?

Reply:
Oh Gosh! I didn't really mean to keep those there XD